06-reactJs/components/movie-list.js

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MovieList = React.createClass({
+
+    render: function () {
+        listNodes = this.props.data;
```

Comment:
As a tip to have cleaner code: when there's logic in your render method, maybe it'll be good to abstract that to a method, e.g.:
```
render: function () {
    {this.renderMovie}
},

renderMovie: function () {
    movieData = this.props.data;

    return (
         // Magic here.
     );
}
```

Also, don't forget to set your ``propTypes``.